test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentsControllerTest < ActionController::TestCase
+  # test "the truth" do
```

Comment:
Cool, we can add another ticket once we close the other one out to writes tests for this feature.